client/layouts/default.vue

```diff
@@ -100,7 +100,10 @@
     </v-app-bar>
     <v-main>
       <div class="pa-s">
-        <nuxt keep-alive :keep-alive-props="{ max: 10 }" />
+        <nuxt
+          keep-alive
+          :keep-alive-props="{ max: 10, exclude: ['fullscreen-playback'] }"
```

Comment:
Non blocking and super nitpicky, but maybe it's not cleaner to set the passed object in ``data`` (now that we're expanding it) for maintainability?

Also, I think you meant fullscreen/playback instead?

Reply:
> Non blocking and super nitpicky, but maybe it's not cleaner to set the passed object in data (now that we're expanding it) for maintainability?

Setting it in data kind of implies a level of reactivity, which this doesn't have. I don't think it's necessary here.

> Also, I think you meant fullscreen/playback instead?

No, this is correct. It's the *router name*, not the route itself. You can guess them fairly easily if you know how they're build, otherwise you can look them up in the Vue Dev Tools, in Routing > Routes.